lib/transformer.dart

```diff
@@ -474,12 +475,12 @@ class InitializerData {
 }
 
 // Reads a file list from a barback settings configuration field.
-_readFileList(BarbackSettings settings, String field) {
+List<String> _readFileList(BarbackSettings settings, String field) {
   var value = settings.configuration[field];
   if (value == null) return null;
-  var files = [];
+  List<String> files = [];
   bool error;
-  if (value is List) {
+  if (value is List<String>) {
```

Comment:
I know that type checking will ignore the type argument, and I agree with you that it can get a reader confused. But I thought it was better than an explicit `as List<String>` to be added below and ... is less typing :))))